### PR TITLE
Roll tonic to da66b94839f788a0bffc34fd9bdfef3360af8c18.

### PR DESCRIPTION
**Pull request by rmacnak-google**
Update to moved repository.

Issue https://github.com/flutter/flutter/issues/9998

### CODE REVIEW

**Comment by rmacnak-google** [comment]
@cbracken @Hixie License script changes

File: tools/licenses/lib/main.dart [hunk 1]
```diff
@@ -2205,6 +2205,38 @@ class RepositoryGarnetFidlDirectory extends RepositoryDirectory {
   }
 }
 
+class RepositoryTopazDirectory extends RepositoryDirectory {
+  RepositoryTopazDirectory(RepositoryDirectory parent, fs.Directory io) : super(parent, io);
+
+  @override
+  bool shouldRecurse(fs.IoNode entry) {
+    return entry.name == 'lib'
+        && super.shouldRecurse(entry);
```
Comment (Hixie) [inline tools/licenses/lib/main.dart]: I'd feel much more comfortable with a black-list than a white-list here (and below). A white-list risks that we add code to a new directory and never notice. Worst case with a black list we end up with an extra license. Worst case with a white list we violate a license.

Comment (rmacnak-google) [reply tools/licenses/lib/main.dart]: Whitelist `topaz/lib/tonic` is now blacklist `topaz/tools` and `topaz/shell/third_party/QR-Code-generator`

**Comment by cbracken** [comment]
lgtm

**Pull request merged**